tb/cpu_uvm/tests/test_random_uvm.py

```
@@ -0,0 +1,149 @@
+            dut: Device under test
+            num_instructions: Number of instructions to generate
+            seed: Random seed (None = random)
+        """
+        super().__init__(name, parent, dut)
+        self.num_instructions = num_instructions
+        self.seed = seed
+
+    async def run_phase(self):
+        """Run random instruction test."""
+        self.logger.info(f"Running random test (seed={self.seed}, "
+                        f"instructions={self.num_instructions})")
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)